src/python_be.cc

```diff
@@ -449,11 +322,9 @@ ModelInstanceState::LaunchStubProcess()
   thread_pool_ = std::make_unique<boost::asio::thread_pool>(
       model_state->StateForBackend()->thread_pool_size);
 
-  if (model_state->IsDecoupled()) {
-    decoupled_thread_ = true;
-    decoupled_monitor_ =
-        std::thread(&ModelInstanceState::DecoupledMessageQueueMonitor, this);
-  }
+  queue_monitor_thread_ = true;
+  decoupled_monitor_ =
```

Comment:
Would it make sense to rename it to `queue_monitor` or something to avoid confusion?

Reply:
Sure, updated.